randovania/games/prime1/logic_database/Phendrana Drifts.txt

```diff
@@ -1438,7 +1468,8 @@ Extra - aabb: [169.47212, -954.347, 79.35217, 197.02737, -882.98553, 105.99773]
   > Pickup (Energy Tank)
       Any of the following:
           Shoot Plasma Beam
-          Boost Ball and Charge Beam and Morph Ball and Space Jump Boots and Clip Through Objects (Advanced) and Single Room Out of Bounds (Hypermode) and Jump Off Enemies (Hypermode) and Standable Terrain (Advanced) and Normal Damage ≥ 125 and Shoot Any Beam
+          # https://youtu.be/uxuwknEX68s
```

Comment:
Isn't this possible NSJ? Maybe that's where the hypermode came from originally

Reply:
This might be possible NSJ, but I haven't seen someone do the SW NSJ before and the logic originally required SJ anyway. If a vid of it gets released I'm cool to pile it in.

Reply:
It should be in Max% NBSJ since that's how you break into phen without Varia